machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -82,3 +88,5 @@ include::modules/cpmso-creating-cr.adoc[leveloffset=+1]
 * xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-azure_cpmso-configuration[Sample YAML for configuring Microsoft Azure clusters]
 * xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-nutanix_cpmso-configuration[Sample YAML for configuring Nutanix clusters]
 * xref:../../machine_management/control_plane_machine_management/cpmso-configuration.adoc#cpmso-sample-yaml-vsphere_cpmso-configuration[Sample YAML for configuring VMware vSphere clusters]
+
+* xref:h../../machine_management/control_plane_machine_management/cpmso-configuration#cpmso-sample-yaml-openstack_cpmso-configuration[Sample YAML for configuring {rh-openStack} clusters]
```

Comment:
typo. Would also suggest using the full name (`rh-openstack-first`) here to match the others
```suggestion
* xref:../../machine_management/control_plane_machine_management/cpmso-configuration#cpmso-sample-yaml-openstack_cpmso-configuration[Sample YAML for configuring {rh-openStack} clusters]
```

Reply:
changed